docs/docs/cmd/spo/site/site-appcatalog-list.md

```diff
@@ -0,0 +1,63 @@
+# spo site appcatalog list
+
+List all site collection app catalogs within the tenant
+
+## Usage
+
+```sh
+m365 spo site appcatalog list [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+!!! important
+    To use this command you have to have at least read permissions on the SharePoint root site.
+
+## Examples
+
+List all site collection app catalogs within the tenant
+
+```sh
+m365 spo site appcatalog list
+```
+
+## Response
+
+### Standard response
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "AbsoluteUrl": "https://contoso.sharepoint.com/sites/site1",
+        "ErrorMessage": null,
```

Comment:
For new sites, I'm getting a value `Success` for `ErrorMessage`. Maybe we should replace `null` with `Success` so people know what kind of value they can expect.